services/office-add-in-api/serverless.yml

```diff
@@ -0,0 +1,58 @@
+service: office-add-in-api
+
+plugins:
+  - serverless-bundle
+  - serverless-add-api-key
```

Comment:
I wonder if `serverless-add-api-key` is necessary since we do not use the apiKey at all in this service